dms/__manifest__.py

```diff
@@ -32,20 +33,18 @@
         "views/dms_portal_templates.xml",
     ],
     "assets": {
+        "mail.assets_messaging": [
+            ("include", "mail.assets_core_messaging"),
+            "dms/static/src/models/*.js",
+        ],
         "web.assets_backend": [
-            "dms/static/src/scss/variables.scss",
-            "dms/static/src/scss/file_kanban.scss",
-            "dms/static/src/scss/directory_kanban.scss",
-            "dms/static/src/js/fields/path.js",
-            "dms/static/src/js/views/many_drop_target.js",
-            "dms/static/src/js/views/search_panel.esm.js",
-            "dms/static/src/js/views/file_list_controller.js",
-            "dms/static/src/js/views/file_list_view.js",
-            "dms/static/src/js/views/file_kanban_controller.js",
-            "dms/static/src/js/views/file_kanban_renderer.js",
-            "dms/static/src/js/views/file_kanban_view.js",
+            "dms/static/src/scss/*",
+            "dms/static/src/js/fields/*",
+            "dms/static/src/js/views/*.esm.js",
+            "dms/static/src/js/views/*.xml",
```

Comment:
Is it correct to include it here?

Reply:
Yeah, I think so, or do you have any suggestions?